### PR TITLE
Create contribution documentation.

### PR DESCRIPTION
**Pull request by cypherhoodlum**
Started documenting the contribution workflow. Appends and edits are welcome. Especially on the coding conventions.

### CODE REVIEW

File: documentation/development/contributing.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Contributing to NostrHub
+
+The NostrHub project operates an open contributor model where anyone is welcome to contribute towards development in the form of peer review, documentation, testing and patches.
+Anyone is invited to contribute without regard to technical experience, "expertise", OSS experience, age, or other concern.
+
+# Communications Channels
+
+All communication happens openly, using github issues, PRs, nostr, or the [Telegram group](https://t.me/nostr_protocol). This ensures that hidden information does not exist, which makes contributing much easier. Please post your nostr public key [here](https://github.com/NostrGit/NostrGit/issues/19) to be noticed on nostr by the team.
```
Comment (PeerRich) [inline documentation/development/contributing.md:8]: the telegram is super crowded, maybe remove it and just here?

Comment (cypherhoodlum) [reply documentation/development/contributing.md:8]: Was hoping you would say that ;)

Comment (PeerRich) [reply documentation/development/contributing.md:8]: crowded as in -- hard to keep track of

**Review by PeerRich — APPROVED** [review]
reads good so far!

lets add this: https://github.com/NostrGit/NostrGit#contributing too

File: documentation/development/contributing.md (hunk 1 above)
Comment (pierresisson) [reply documentation/development/contributing.md:8]: agree

Comment (PeerRich) [reply documentation/development/contributing.md:8]: man i cant wish for nostr-based group chat

**Comment by cypherhoodlum** [comment]
I need to fix the branch naming conventions still.

File: documentation/development/contributing.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Contributing to NostrHub
```
Comment (PeerRich) [inline documentation/development/contributing.md:1]:
```suggestion
# Contributing to NostrGit
```

File: documentation/development/contributing.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+The NostrHub project operates an open contributor model where anyone is welcome to contribute towards development in the form of peer review, documentation, testing and patches.
```
Comment (PeerRich) [inline documentation/development/contributing.md:3]:
```suggestion
The NostrGit project operates an open contributor model where anyone is welcome to contribute towards development in the form of peer review, documentation, testing and patches.
```

**Review by PeerRich — APPROVED** [review]

File: documentation/development/contributing.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+To contribute a patch, the worflow is a as follows:
+
+1. Fork Repository
+2. Create topic branch
+    - Always use the naming conventions
+        - username/topic/spesific_topic (eg. cypherhoodlum/documentation/contributing)
+        - username/feature/short_description
+        - username/fix/what_needs_fixing
+        - etc.
```
Comment (PeerRich) [inline documentation/development/contributing.md:22]: I would recommend creating branches from within issues

Comment (cypherhoodlum) [reply documentation/development/contributing.md:22]: Agree.

**Review by PeerRich — APPROVED** [review]

**Comment by PeerRich** [comment]
merginggggg

**Pull request merged**